app/services/Athena.scala

```diff
@@ -0,0 +1,177 @@
+package services
```

Comment:
the downside to athena is that executing queries is very complicated!

Reply:
Naive question: how "live" is the extracted data? Or, to put it another way, how often does the query run?

Reply:
I think the data used by athena comes through within 5 mins (there's some batching before it gets written to s3).
SAC then uses athena to query that data when a user double clicks a row in this page

Reply:
Is it worth considering a cache for this data - say 10mins? - to prevent excessively running the query when a user goes mad and clicks on several articles in a matter of seconds?

Reply:
the query is different for each article/time. I'm not too worried about what our users can do here - athena is cheap! On the hack day when I was testing this repeatedly I spent $0.20